floss/language/go/extract.py

```diff
@@ -0,0 +1,318 @@
+# Copyright (C) 2023 Mandiant, Inc. All Rights Reserved.
+
+import re
+import sys
+import struct
+import logging
+import argparse
+from typing import List, Iterable, Optional
+from pathlib import Path
+from itertools import chain
+
+import pefile
+
+from floss.main import get_static_strings
+from floss.results import StaticString, StringEncoding
+
+logger = logging.getLogger(__name__)
+
+MIN_STR_LEN = 6
+
+
+def extract_stackstrings(extract_stackstring_pattern, section_data, min_length) -> List[StaticString]:
```

Comment:
can you compare the results here to our existing FLOSS stackstrings algorithm? maybe we can just that